genx/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "{{groupId}}.pbc-{{appName}}-seed",
```

Comment:
I think we need one here as well, because here we want something like 'https://github.com/genesislcap', whereas I think this group id is more in the java standard way of com.mycompany ?

Reply:
. -> /